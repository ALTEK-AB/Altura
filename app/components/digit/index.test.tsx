import {render} from '@testing-library/react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import Digit from './';
// expect.extend({toHaveStyle});
describe('<Digit />', () => {
  test('should render a <View /> component', () => {
    const {getByTestId} = render(<Digit isSelected={false} />);
    const digitComponent = getByTestId('digit-component');
    expect(digitComponent).toBeDefined();
    expect(digitComponent.type).toEqual('View');
  });

  test('renders correctly', () => {
    const tree = renderer.create(<Digit isSelected={true} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
