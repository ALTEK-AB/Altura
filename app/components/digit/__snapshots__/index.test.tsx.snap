// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Digit /> renders correctly 1`] = `
<View
  style={
    [
      {
        "borderRadius": 5,
        "height": 10,
        "width": 10,
      },
      {
        "backgroundColor": "#949BC3",
      },
    ]
  }
  testID="digit-component"
/>
`;
